pkg/util/cluster/cluster.go

```diff
@@ -152,7 +152,7 @@ func (c *Cluster) CreateApp(ctx context.Context, clusterName string) error {
 		return err
 	}
 
-	return os.WriteFile("clusterapp.env", []byte(fmt.Sprintf("AZURE_CLUSTER_SERVICE_PRINCIPAL_ID=%s\nAZURE_CLUSTER_APP_ID=%s\nAZURE_CLUSTER_APP_SECRET=%s", spID, appID, appSecret)), 0o600)
+	return os.WriteFile("clusterapp.env", []byte(fmt.Sprintf("export AZURE_CLUSTER_SERVICE_PRINCIPAL_ID=%s\nexport AZURE_CLUSTER_APP_ID=%s\nexport AZURE_CLUSTER_APP_SECRET=%s", spID, appID, appSecret)), 0o600)
```

Comment:
Now that we are touching this line, it could be a good idea to make a small change and replace this single line with something easier to read (the current line is long and contains multiple parentheses)
```go
content := fmt.Sprintf("export AZURE_CLUSTER_SERVICE_PRINCIPAL_ID=%s\nexport AZURE_CLUSTER_APP_ID=%s\nexport AZURE_CLUSTER_APP_SECRET=%s", spID, appID, appSecret)
contentBytes := []byte(content)
return os.WriteFile("clusterapp.env", contentBytes, 0o600)
```